doc/source/tokudb/tokudb_variables.rst

```diff
@@ -0,0 +1,432 @@
+.. _tokudb_variables:
+
+================
+TokuDB Variables
```

Comment:
Yup, you're right, and I looked right at them, said, good they are here, got distracted for a minute, then carried on reporting them as missing...sorry :-\